keyboards/momoka_ergo/readme.md

```diff
@@ -0,0 +1,19 @@
+# momoka_ergo
+
+![momoka_ergo](https://github.com/StefanGrindelwald/TestDemo/blob/master/Momoka_ergo.jpg?raw=true)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [StefanGrindelwald](https://github.com/StefanGrindelwald)
+* Hardware Supported: momoka.co/ergo
+* Hardware Availability: momoka.co/ergo
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add information on how to jump into the bootloader to flash the board

Reply:
Thanks, I just added this.